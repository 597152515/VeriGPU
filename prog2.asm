out abx
out cdx
out 12x
out 34x
