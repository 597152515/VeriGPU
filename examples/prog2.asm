out abx
out cdx
out 12x
out 34x
halt

10x:
word 0000deadx
word 0000beefx
