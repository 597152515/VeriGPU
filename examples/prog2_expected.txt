out 0 00000000000000000000000001111011 0000007b 123
out 1 00000000000000000000001100010101 00000315 789
out 2 00000000000000000000000000110010 00000032 50
out 3 00000000000000000000000001000100 00000044 68
out 4 00000000000000000000000000110010 00000032 50
out 5 00000000000000000000000000111100 0000003c 60